integreat_cms/static/src/css/style.scss

```
@@ -537,7 +537,38 @@ datalist option {
 }
 
 /* Special word break for feedback list */
-.feedback-comment-full {
-  @apply break-words whitespace-pre-line;
-  word-break: break-word;
+.feedback-entry {
+  display: fleX;
+  flex-wrap: nowrap;
+  span {
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: no-wrap;
+  }    
+  
+  .less {
+    display: none;
+  }
+
+  .more {
+    display: inline;
+  }
+
+  &.active {
+    flex-wrap: wrap;
+
+    span {
+      @apply break-words whitespace-pre-line;
+      word-break: break-word;
+      overflow: visible;
+    }
+
```

Optional - but I think this looks nicer, since the icons are kept centered next to the expanded text:

```suggestion
.feedback-entry {
  display: flex;
  flex-flow: row nowrap;
  align-items: center;
  
  span {
      overflow: hidden;
      text-overflow: ellipsis;
      white-space: no-wrap;
  }
  
  a {
    padding: 0 0.1rem;
  }    
  
  .less {
    display: none;
  }

  .more {
    display: inline;
  }

  &.active {
    span {
      @apply break-words whitespace-pre-line;
      word-break: break-word;
      overflow: visible;
      white-space: initial;
    }

```

---

Whoops, I don't know why the diff is so large. Important lines are 541-543, 551-553, and 568.